index.html

```diff
@@ -71,12 +71,12 @@ <h2 class="title">Language</h2>
         <nav class="footer-nav">
           <ul class="footer-nav-list">
             <li class="footer-nav-list"><a href="https://github.com/Yuliya241" class="nav-link"><img src="assets/image/github.ico" width=30 alt="github"></a></li>
-            <li class="footer-nav-list"><a href="https://rs.school/js-stage0/" class="nav-link"><img src="assets/image/rs school JS.ico" width=30 alt="rs school"></a></li>
+            <li class="footer-nav-list"><a href="https://rs.school/js-stage0/" class="nav-link"><img src="assets/image/rsschoolJS.ico" width=30 alt="rs school"></a></li>
```

Comment:
It's better to use lowercase for file names. 

`rsschoolJS.ico` => `rsschool-js.ico`

`width=30` => `width="30px"`